### ISSUE TITLE
#11 Simplify Loss Calculation: Create a Function for Loss Calculation

### ISSUE THREAD

**Issue by shresth-keshari** [veeradi34/Hello-foss-Video-summarization#14]
Description: The calculation of the discriminator and generator losses in the training loop is verbose. This can be simplified by creating a dedicated function.

Tasks:

Create a function calculate_losses(real_validity, fake_validity, criterion) that returns the losses for both the discriminator and generator.
Update the training loop to call this function.
